### ISSUE TITLE
Site 404 on clicking [back]

### ISSUE THREAD

**Issue by Pross** [wpdirectory/wpdir#4]
To reproduce simply navigate to https://wpdirectory.net/about then click on one of the links, Github for example.
Now click back button in the browser. You will get `404 page not found` from the app.

**Comment by PeterBooker** [comment]
Thank you for the report. There is a mismatch between the URL formats of React and the web server. React does not need a backslash at the end of the URL, but the web server does. I should be able to fix this quickly.

**Comment by PeterBooker** [comment]
This should be resolved now. It was a little more complex than I thought. I had removed the `/static` prefix from the fileserver, which had caused the not found handler to stop working unknown to me. So in resolving the above issue, redirecting all URLs with a trailing slash to no trailing slash, I also fixed the not found handler, so any unknown URLs still lead to the React front-end.

Thank you very much for the report, it uncovered far more problems than I expected and has helped to resolve many issues.
